eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -255,7 +272,10 @@ stages:
                 runOnce:
                   deploy:
                     steps:
-                      - checkout: self
+                      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+                        parameters:
+                          Paths:
+                            - '**/*.xml'
```

Comment:
Please have a closer look at this one I think you might also need *.md files as the version increment also generally updates changelog.md files.

Reply:
The package changelog gets checked out as part of the service directory path included in the `Paths` parameter.

I generally test template release with each functional commit, here's the most recent passing run: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=909741&view=logs&s=d0889692-cae7-51e0-3e1f-b1d8c62a4117&j=ea2ca241-627c-528c-676e-0ff517e2e7f6